components/raftstore/src/store/snap.rs

```diff
@@ -597,7 +597,11 @@ impl Snapshot {
             }
 
             if !plain_file_used(cf_file.cf) {
-                // Reset global seq number.
+                // TODO: remove reset_global_seq() once write_global_seqno is only allowed to be set
```

Comment:
Do you decide to keep the `reset_global_seq` logic for backward compatiblity? If so the inline comment need some update. And shall we add a test for the backward compatiblity?

Reply:
> Could you please be more specific on what type of test you want see?
> 
> As for the compatibility issue, I listed my observations below.

basically the `test_ingest_reentrant` test, but use `write_global_seqno=true` in the first ingestion.